tools/roslaunch/test/unit/test_roslaunch_dump_params.py

```diff
@@ -77,6 +77,10 @@ def test_roslaunch(self):
             '/node_rosparam/dict1/shoulders': 2,
             '/node_rosparam/dict1/knees': 3,
             '/node_rosparam/dict1/toes': 4,
+            '/node_rosparam/tilde1': 'foo',
+            '/node_rosparam/local_param': 'baz',
+
+            '/node_rosparam2/tilde1': 'foo',
```

Comment:
Is there a way to do a negative test here, eg:

```
    '/node_rosparam2/tilde2': None
```

Might need something changed in the python file to correctly handle it.

Reply:
I think the equality tests (`val == output_val` and `assertEquals(val, output_val)`) already covered that case. However, for clarity, I added the negative tests at the bottom.